user_docs/en/changes.md

```diff
@@ -13,6 +13,7 @@
 
 ### Bug Fixes
 
+* Fixed missing MathML attributes for `mspace` in PDF documents. Also passed along attributes for elementary math elements.
```

Comment:
could you please rewrite this to be user facing, i.e. explain how behaviour has changed in NVDA.
please also add a reference to the PR number, and your GitHub username if you wish

Reply:
Done.